tests/models/test_default_evaluator.py

```diff
@@ -104,13 +190,56 @@ def test_regressor_evaluation(linear_regressor_model_uri, diabetes_dataset):
         "shap_feature_importance_plot_on_data_diabetes_dataset.png",
         "shap_summary_plot_on_data_diabetes_dataset.png",
     }
+
     assert result.artifacts.keys() == {
         "shap_beeswarm_plot",
         "shap_feature_importance_plot",
         "shap_summary_plot",
     }
 
 
+def test_regressor_evaluation_disable_logging_metrics_and_artifacts(
+    linear_regressor_model_uri,
+    diabetes_dataset,
+):
+    with mlflow.start_run() as run:
+        result = evaluate_model_helper(
+            linear_regressor_model_uri,
+            linear_regressor_model_uri,
+            diabetes_dataset._constructor_args["data"],
+            model_type="regressor",
+            targets=diabetes_dataset._constructor_args["targets"],
+            dataset_name=diabetes_dataset.name,
+            evaluators="default",
+            eval_baseline_model_only=True,
+        )
+
+    _, metrics, tags, artifacts = get_run_data(run.info.run_id)
+
+    model = mlflow.pyfunc.load_model(linear_regressor_model_uri)
+
+    y = diabetes_dataset.labels_data
+    y_pred = model.predict(diabetes_dataset.features_data)
+
+    expected_metrics = _get_regressor_metrics(y, y_pred)
+    expected_metrics["score"] = model._model_impl.score(
+        diabetes_dataset.features_data, diabetes_dataset.labels_data
+    )
+
+    check_metrics_not_logged_for_baseline_model_evaluation(
+        expected_metrics=expected_metrics,
+        result_metrics=result.metrics,
```

Comment:
Should this be baseline metrics instead?

Reply:
no, we technically treated baseline model as "candidate_model`

Reply:
In that case, we aren't testing the candidate model code path, which defeats the purpose of the test.

Reply:
Update to baseline metrics for all disable logging tests